packages/plans-grid/src/badge/index.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import classNames from 'classnames';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface Props {
+	children: React.ReactElement[];
+	className?: string;
+}
+
+const Badge: React.FunctionComponent< Props > = ( { children, className, ...props } ) => (
+	<span { ...props } className={ classNames( 'badge', className ) }>
+		{ children }
+	</span>
+);
+
+export default Badge;
```

Comment:
Ok to include here in this PR, but let's either move it to `@automattic/components` or copy over the badge component from Calypso to `@automattic/components` and us that instead.

Reply:
I've added some @TODOs and created #42950 to keep track of these and move them one by one to packages.